### PR TITLE
add text to SVGs and update actor definitions

### LINKED ISSUE #120: Network Value Chain
In the "network value chain" page the concepts are well defines but it doesn't reflect that there can be more than one network provider. The definition of network provider talks of "The network provider" and the diagram does not show multiple network providers. Its now common to have multiple network providers in infra sharing contracts where multiple providers take multiple fibre strands. Normally there is only one physical infrastructure provider though, but this can be an SPV co owned by the network providers.

### LINKED ISSUE #138: SVGs should contain translatable strings
The SVG diagrams we use across the site were made using Google draw and exported into SVG format. This conversion process does not render text as translatable strings. As we may wish to translate these diagrams in the future we should use a process for creating diagrams that enables this, and update existing diagrams so that they contain translatable strings. We outlined a process for this for the [beneficial ownership data standard](https://openownership.github.io/bods-dev-handbook/diagram_creation.html).

I'll create a separate issue around documentation in the OFDS dev handbook.

### PR DESCRIPTION
**Pull request by lgs85**
Closes #138 
Closes #120

### CODE REVIEW

File: docs/primer/scopeandkeyconcepts.md [hunk 1]
```diff
@@ -22,25 +22,21 @@ The primary focus of OFDS is to describe the passive network infrastructure. The
 
 ### Actors
 
-Based on the layers in the network value chain, there are three main actors in a fibre optic network:
+Based on the layers in the network value chain, there are three main groups of actors in a fibre optic network:
 
 ![The actors in fibre optic networks](../../_assets/actors.svg)
```
Comment (duncandewhurst) [inline docs/primer/scopeandkeyconcepts.md:28]: Add a clarifying sentence to further emphasise the plural nature of each role:

```suggestion

Depending on the [business model](#network-business-models) used in a network, there can be one or more of each type of actor involved in a single network.

```

File: docs/primer/scopeandkeyconcepts.md [hunk 2]
```diff
@@ -22,25 +22,21 @@ The primary focus of OFDS is to describe the passive network infrastructure. The
...
 
 #### Physical infrastructure provider
 
 OFDS defines a physical infrastructure provider as:
 
-```{jsoninclude-quote} ../../schema/network-schema.json
-:jsonpointer: /definitions/Link/properties/physicalInfrastructureProvider/description
-```
+>An organisation that owns and maintains passive network infrastructure, i.e. the non-electrical elements, such as dark fibre, ducts and physical sites.
```
Comment (duncandewhurst) [inline docs/primer/scopeandkeyconcepts.md:35]: Hardcoding the code descriptions is fine for this PR but creates a SSOT issue going forward. I've opened an issue: https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/142

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
* `actors.svg`: Remove the initialisms (SP, NP, PIP) as these aren't referenced anywhere.
* `networkBusinessModels.svg`:
  * There should only be one network provider in the active-layer open model (per the original diagram). It's the three-layer open model that needs updating to show multiple network providers. I think we'll need to make the diagram a bit wider so that the three-layer open model includes a network provider with two service providers and another network provider with one service provider.
  * Let's split the physical infrastructure provider box in two so that it doesn't span two different business models

Once the changes are made, we should share the branch preview with Ben to check that it addresses his concerns.

**Comment by duncandewhurst** [comment]
Please update the changelog, too. https://github.com/Open-Telecoms-Data/open-fibre-data-standard/pull/144 adds some structure to the changelog so it would be good to merge that first and then to merge 0.1-dev into this branch, to avoid conflicts.

File: docs/primer/scopeandkeyconcepts.md (hunk 2 above)
Comment (lgs85) [reply docs/primer/scopeandkeyconcepts.md:35]: yep good plan

**Review by duncandewhurst — APPROVED** [review]

**Pull request merged**